core/src/main/scala/anorm/Column.scala

```diff
@@ -276,6 +276,12 @@ object Column extends JodaColumn with JavaTimeColumn {
     val MetaDataItem(qualified, _, _) = meta
     value match {
       case bool: Boolean => Right(bool)
+      case bit: Byte if bit == 1 => Right(true)
```

Comment:
As said on the issue, I'm not for having that MariaDB specific behaviour in the common code base.

Reply:
Well you are the maintainer here, so if you are certain of the decision, then this pull request and the associated issue can be closed.

Reply:
Rather a separate non-default `Column` instance, in `features`, with explicit documentation about MariaDB usage

Reply:
I had a bit of trouble finding what `features` you were talking about, as it is currently an empty object. I dug up the 2.4 release tag to have an example. Hopefully they way I did it in latest version is what you intended.